src/BSON/Timestamp.c

```diff
@@ -67,6 +67,48 @@ static bool php_phongo_timestamp_init(php_phongo_timestamp_t *intern, phongo_lon
 	return true;
 }
 
+/* Initialize the object from numeric strings and return whether it was
+ * successful. An exception will be thrown on error. */
+static bool php_phongo_timestamp_init_from_string(php_phongo_timestamp_t *intern, const char *s_increment, phongo_zpp_char_len s_increment_len, const char *s_timestamp, phongo_zpp_char_len s_timestamp_len TSRMLS_DC)
+{
+	int64_t increment, timestamp;
+	char *endptr = NULL;
+
+	errno = 0;
+
+	/* errno will set errno if conversion fails; however, we do not need to
```

Comment:
UTCDateTime.c had this comment block above the `if (errno || ...` lines below. Any reason you decided to move it up?